src/layouts/default.vue

```diff
@@ -92,6 +93,10 @@
     },
 
     mounted() {
+      this.$announcer.setComplementRoute(this.$t('pageHasLoaded'));
```

Comment:
I don't know, it was there in PR #920, but got removed. Because the Klaro branch is an older branch, I can imagine it is leftover from a merge? Since it is not on the master branch and it is indeed working with, I have removed the line.